### ISSUE TITLE
Failed to unpack inbound alert request

### ISSUE THREAD

**Issue by acmodeu** [benjojo/alertmanager-discord#24]
I want to send some alerts from external exporter via kubernetes prometheus stack to discord. I see the alerts in the alertmanager but they don't appear in discord channel and there are a lot of similar errors in alertmanager-discord logs.

```
2021/08/12 19:46:39 10.200.129.205:9094 - [GET] 
2021/08/12 19:46:39 Failed to unpack inbound alert request - 
```

**Comment by benjojo** [comment]
Are you sure the alert manager is actually sending the alert webhooks to
this software?

Those logs you provided look like they are just health checks, presumably
from k8s

**Comment by acmodeu** [comment]
Yes. I'm sure, I set up discord as default receiver for all alerts but received all other kubernetes alerts except those external ones. I switched to telegram bot and received them right away.

**Comment by benjojo** [comment]
So, alertmanager webhooks come in as POST's so you should look for something like

```
2021/08/12 19:46:39 10.200.129.205:9094 - [POST] 
```

in your logs, if you can't find them, webhooks are not directly landing on alertmanager-discord.

**Comment by h4wkmoon** [comment]
Hi, 

When using into k8s, you may want to use readiness and liveness probes.
They usually make GET requests to the app, to see if the process is alive and fit for duty. If something other than 2xx is returned, kubernetes will kill the container, and spawn another one.

**Comment by HamidullahMuslih** [comment]
same issue on our side, i think the latest alert manager service payload can't be unpacked by the current version of this service.
 
we have published the container port as `-p 9098:9094` and below are the logs

alertmanager logs: 
```
ts=2024-12-13T15:07:50.793Z caller=notify.go:848 level=warn component=dispatcher receiver=discord_webhook_eng integration=webhook[0] aggrGroup="{}/{alertname=~\"HostOutOfMemory|HostOomKillDetected|"}" msg="Notify attempt failed, will retry later" attempts=1 err="Post \"<redacted>\": read tcp 127.0.0.1:52310->127.0.0.1:9098: read: connection reset by peer"
```

discord service logs
```
2024/12/13 14:11:38 Listening on: 0.0.0.0:9094
2024/12/13 14:38:38 <ip>:9098 - [GET]
2024/12/13 14:38:38 Failed to unpack inbound alert request -
```
